src/components/molecules/Button/styles.module.scss

```diff
@@ -0,0 +1,117 @@
+@import 'src/styles/variables';
+@import 'src/styles/mixins';
+
+.baseButton {
+  display: flex;
+}
+
+.buttonText {
+  order: 2;
+  @include buttonTextStyle;
+  line-height: $line-height-24;
```

Comment:
**[blocker]**

Unnecessary new line-height. The correct line-height was already applied. Most text in this project has a line-height of 1.5, which is applied globally and is also true for buttonText.

buttonTextStyle has font-size of 16px. This means that line-height: 1.5 means we multiply 16 by 1.5, which is 24px.